src/libs/state.js

```diff
@@ -15,6 +15,14 @@ export const authStore = Utils.atom({
   cookiesAccepted: undefined
 })
 
+export const userStatus = {
+  unregistered: 'unregistered',
+  createdProfile: 'createdProfile',
+  registeredWithoutTos: 'registeredWithoutTos',
```

Comment:
What's the difference between `registeredWithoutTos` and `createdProfile`? In my mental model, a user that has registered but hasn't accepted the ToS is a user that has only created a profile in Terra.